pom.xml

```diff
@@ -130,5 +131,52 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-stream</artifactId>
+            <version>${spring-cloud-stream.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.aspectj</groupId>
+            <artifactId>aspectjweaver</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.powsybl</groupId>
+            <artifactId>powsybl-commons</artifactId>
+        </dependency>
```

Comment:
Done

Reply:
Done

Reply:
Done.

Reply:
Done

Reply:
Idem as i mentioned above, in the future we should create a TS to avoid transitive dependencies as suggested by Maven: https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html

> Although transitive dependencies can implicitly include desired dependencies, it is a good practice to explicitly specify the dependencies your source code uses directly.